operations/mimir-mixin/config.libsonnet

```
@@ -272,7 +272,7 @@
             sum by (%(alert_aggregation_labels)s, deployment) (
               label_replace(
                 label_replace(
-                  node_namespace_pod_container:container_cpu_usage_seconds_total:sum_irate,
+                  sum by (%(alert_aggregation_labels)s, namespace, %(per_instance_label)s, container)(rate(container_cpu_usage_seconds_total[1m])),
```

namespace looks redundant here, because already part of `alert_aggregation_labels`:

```suggestion
                  sum by (%(alert_aggregation_labels)s, %(per_instance_label)s, container)(rate(container_cpu_usage_seconds_total[1m])),
```